### DISCUSSION TITLE
trouble installing plugins

### DISCUSSION THREAD

**Discussion by demiscuzz** [x64dbg/x64dbg#2675] [Q&A]
Hi all, I am following a guide on using x64dbg and it tells me to drop my plugin into the plugin folder, but there is no plugin folder on the latest release. I have tried creating the folder "plugins" and pasting them there but that didnt work either. Am i missing a trick. 
I am installing on windows 7 vm, downloaded the latest release
the plugin that i am trying to install is ERC.Xdbg_32-1.4

Thanks

**Comment by mrfearless** [comment]
The plugins folder is _X_`:\x64dbg\x32\plugins` (for x86 version of x64dbg) or _X_`:\x64dbg\x64\plugins` (for x64 version of x64dbg), where _X_ is the drive letter where you installed x64dbg. You can manually create these if needed.

**Reply by demiscuzz** [reply]
Thank you for clarifying that, I have created the plugins folder, copied my plugin over and it now appears in x32dbg. All working fine. 
Cheers

**Reply by ozzy42037317** [reply]
how do u manually create it
